dspace/bin/make-handle-config

```diff
@@ -46,23 +53,24 @@ echo "n" >>$tempfile  # Do not encrypt admin private key
                       # OR do not overwrite old admin private key
 
 # Now run Handle server config tool, and pipe our config in
-$BINDIR/dspace dsrun net.handle.server.SimpleSetup $handledir < $tempfile >/dev/null
+$BINDIR/dspace dsrun net.handle.server.SimpleSetup $handledir < $tempfile >/dev/null 2>&1
 
 # Remove temp file
 rm -f $tempfile
 
 # Now update server configuration (config.dct) with our handle prefix and the
 # DSpace Handle plugin
-sed 's/YOUR_NAMING_AUTHORITY/'$handle'/' <$handledir/config.dct >$tempfile
+sed 's/YOUR_PREFIX/'$handle'/' <$handledir/config.dct >$tempfile
```

Comment:
I think keeping it `YOUR_NAMING_AUTHORITY` makes sense, since that matches the install documentation for 6x?

Reply:
That line is updating the default Handle config file, which uses `YOUR_PREFIX` in Handle 9. So, that change is correct, but means the DSpace 6 docs won't match anymore.